prow.sh

```diff
@@ -1066,15 +1066,21 @@ main () {
                 docker tag "$i:latest" "$i:csiprow" || die "tagging the locally built container image for $i failed"
             done
 
-            if [ -e deploy/kubernetes/rbac.yaml ]; then
+            if [ -e "$(pwd)/deploy/kubernetes/rbac.yaml" ] || [ -e "$(pwd)/deploy/kubernetes/csi-snapshotter/rbac-csi-snapshotter.yaml" ]; then
```

Comment:
Why the additional `$(pwd)`? That's redundant here.

Reply:
I think this doesn't quite work for snapshotter repo which has two binaries with two rbac files.

Maybe we need to recursively search for rbac files and apply all of them?

Reply:
I thought the $(pwd) was needed for this test: `pull-kubernetes-csi-external-snapshotter-1-16-on-kubernetes-1-16`. This was failing to hit this conditional originally. Let me remove it temporarily and see if that's still the case still.

Snapshotter 1.x rbac file is always `deploy/kubernetes/rbac.yaml`. The reason for needing an additional check here is for 2.x compatibility, where the csi-snapshotter RBAC is here: `deploy/kubernetes/csi-snapshotter/rbac-csi-snapshotter.yaml`

Regarding the snapshot-controller RBAC, it is taken care of here separately:
https://github.com/kubernetes-csi/csi-release-tools/blob/master/prow.sh#L1089
https://github.com/kubernetes-csi/csi-release-tools/blob/master/prow.sh#L727

Reply:
I was mistaken, looks like $(pwd) isn't needed. Will remove now.
https://prow.k8s.io/view/gcs/kubernetes-jenkins/pr-logs/pull/kubernetes-csi_external-snapshotter/299/pull-kubernetes-csi-external-snapshotter-1-16-on-kubernetes-1-16/1250813186752385026

Reply:
ok, the problem was actually the quotations around `deploy/kubernetes/rbac.yaml`, not the $(pwd). I just removed the quotations and it did not enter the conditional.